src/sentry/api/endpoints/group_tags.py

```diff
@@ -12,12 +12,14 @@
 
 class GroupTagsEndpoint(GroupEndpoint):
     def get(self, request, group):
+        grouptagkeys = list(GroupTagKey.objects.filter(
```

Comment:
yeah this won't work as it could be like a billion rows

Reply:
Is there a place off the top of your head where we already handle iterating across different DBs?

Reply:
oh actually this is fine, grouptagkeys is limited to one group, sorry!